packages/react-aria-components/src/ColorWheel.tsx

```diff
@@ -67,7 +66,7 @@ export const ColorWheel = forwardRef(function ColorWheel(props: ColorWheelProps,
 });
 
 export interface ColorWheelTrackRenderProps extends ColorWheelRenderProps {}
-export interface ColorWheelTrackProps extends StyleRenderProps<ColorWheelTrackRenderProps>, DOMProps {}
+export interface ColorWheelTrackProps extends StyleRenderProps<ColorWheelTrackRenderProps> {}
```

Comment:
just to double check: was looking through the RAC changes and noticed we also added DOMProps to OverlayArrow, I guess we want to keep that instance of it since OverlayArrow can be used in multiple components rather than ColorWheelTrack only being in a single one?